models/model.js

```diff
@@ -59,32 +60,29 @@ exports.fetchReviewById = (review_id) => {
 };
 
 exports.provideReviewComment = (review_id, newComment) => {
-  console.log(review_id, newComment);
-
   if (
     newComment.hasOwnProperty("username") === false ||
     newComment.hasOwnProperty("body") === false
   ) {
     return Promise.reject({ status: 400, msg: "bad request" });
   }
 
-  db.query(
-    `INSERT INTO users
-  (username, name)
-  VALUES
-  ($1, $1)RETURNING*;`,
-    [newComment.username]
-  );
+  if (review_id > reviews.length) {
```

Comment:
Not a good way to do this, say the test data changes or has staggered ids. You've used a better solution in a previous ticket.